en_ewt-ud-dev.conllu

```diff
@@ -20528,17 +20528,17 @@
 1	If	if	SCONJ	IN	_	6	mark	6:mark	_
 2-3	your	_	_	_	_	_	_	_	_
 2	you	you	PRON	PRP	Case=Nom|Person=2|PronType=Prs	6	nsubj	6:nsubj|8:nsubj	_
-3	r	be	AUX	VBP	Mood=Ind|Number=Sing|Person=2|Tense=Pres|VerbForm=Fin	6	cop	6:cop	_
+3	r	be	AUX	VBP	Mood=Ind|Number=Sing|Person=2|Tense=Pres|Typo=Yes|VerbForm=Fin	6	cop	6:cop	CorrectForm=are
 4	in	in	ADP	IN	_	6	case	6:case	_
 5	the	the	DET	DT	Definite=Def|PronType=Art	6	det	6:det	_
 6	USA	USA	PROPN	NNP	Number=Sing	13	advcl	13:advcl:in	_
 7	and	and	CCONJ	CC	_	8	cc	8:cc	_
-8	venture	venture	VERB	VB	VerbForm=Inf	6	conj	6:conj:and|13:advcl:in	_
+8	venture	venture	VERB	VB	Mood=Ind|Number=Sing|Person=2|Tense=Pres|VerbForm=Fin	6	conj	6:conj:and|13:advcl:in	_
 9	into	into	ADP	IN	_	10	case	10:case	_
 10	Cuba	Cuba	PROPN	NNP	Number=Sing	8	obl	8:obl:into	_
 11	then	then	ADV	RB	PronType=Dem	13	advmod	13:advmod	_
-12	u	u	PRON	PRP	_	13	nsubj	13:nsubj	_
-13	need	need	VERB	VBP	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	0:root	_
+12	u	you	PRON	PRP	Case=Nom|Person=2|PronType=Prs|Typo=Yes	13	nsubj	13:nsubj	CorrectForm=you
```

Comment:
Instead of `Typo=Yes` I'd say `Abbr=Yes`

Reply:
OK